packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -343,9 +407,9 @@ try {
     const message = event.data;
     switch (message.type) {
       case 'didUpdateGraphData':
-        const graphData = augmentGraphInfo(message.payload);
-        console.log('didUpdateGraphData', graphData);
-        Actions.refresh(graphData);
+        model.graphData = augmentGraphInfo(message.payload);
```

Comment:
I don't think we should touch the `model` here. The model should only be touched by `Actions` and the `update` fn so we keep the unidirectional data flow - am I missing something?

Reply:
No. I'm still working out MVC. I'll get it though.

Reply:
See my [comment above](https://github.com/foambubble/foam/pull/737#discussion_r692994457), let's not use MVC, let's use a unidirectional data flow pattern.